### PR TITLE
ginkgo: Get rid of K8sUpdates

### PR DESCRIPTION
**Pull request by brb**
It got replaced by ci-e2e-upgrade.

### CODE REVIEW

**Review by bimmlerd — APPROVED** [review]

**Review by brlbil — APPROVED** [review]

**Comment by brb** [comment]
https://github.com/cilium/cilium/blob/main/cilium-cli/connectivity/tests/errors.go#L56 is out of sync with https://github.com/cilium/cilium/blob/main/test/helpers/cons.go#L312. Since the cilium-cli got merged, we can just use one list. Converting this PR to draft until it has been resolved.

**Review by ldelossa — APPROVED** [review]
LGTM

File: .github/workflows/conformance-k8s-kind.yaml [hunk 1]
```diff
@@ -196,6 +196,16 @@ jobs:
             --report-dir=${E2E_REPORT_DIR}                \
             --disable-log-dump=true
 
+      - name: Poor man's file-descriptor leak detector
+        run: |
+          for p in $(pidof cilium-agent); do
+            num=$(sudo lsof -p $p | wc -l)
+            if [ "$num" -gt "500" ]; then
+              echo "cilium-agent($p) has more than 500 file descriptors (potential leak)"
```
Comment (gandro) [inline .github/workflows/conformance-k8s-kind.yaml]: Apologies for the drive-by comment  - but if implementable I think this would be a good Prometheus metric to have (which we could then also assert in `cilium connectivity check` in all workflows)

Comment (brb) [reply .github/workflows/conformance-k8s-kind.yaml]: I moved the commits to https://github.com/cilium/cilium/pull/36241.

**Review by aanm — APPROVED** [review]
